extensions/vscode/src/autocomplete/statusBar.ts

```diff
@@ -47,20 +47,20 @@ const statusBarItemText = (
   error?: boolean,
 ) => {
   if (error) {
-    return "$(alert) Continue (FATAL ERROR)";
+    return "$(alert) Granite.Code (FATAL ERROR)";
```

Comment:
maybe use a constant everywhere for the name. 
Might make sense to add an EXTENSION_LABEL constant in core/control-plane/env.ts and upstream the change

Reply:
I don't think it makes sense *unless* we can upstream it, since otherwise we create double edits (one to import, one to use). If we can upstream it, yeah, it would be better.